adcm_client/wrappers/api.py

```diff
@@ -117,10 +117,15 @@ def auth(self, username, password):
             scheme='Token',
             token=token['token']
         )
+        self.api_token = token['token']
```

Comment:
I think it is better to move the definition of this variable above and refer to it on line 118

Reply:
Done.